modules/programs/mcfly.nix

```diff
@@ -81,6 +92,11 @@ in {
 
     (mkIf cfg.enableLightTheme { home.sessionVariables.MCFLY_LIGHT = "TRUE"; })
 
-    (mkIf cfg.enableFuzzySearch { home.sessionVariables.MCFLY_FUZZY = "TRUE"; })
+    (mkIf cfg.enableFuzzySearch {
+      home.sessionVariables.MCFLY_FUZZY = "TRUE";
+    })
+    (mkIf (!cfg.enableFuzzySearch && cfg.fuzzySearchFactor) {
+      home.sessionVariables.MCFLY_FUZZY = "${toString cfg.fuzzySearchFactor}";
+    })
```

Comment:
We only use the new configuration flag is the deprecated one is not set. 
Both unset still results in 0/false/no fuzziness.